src/test/ui/associated-type-bounds/issue-61752.rs

```diff
@@ -0,0 +1,24 @@
+// run-pass
```

Comment:
```suggestion
// check-pass
```
Same; you never test the dynamic semantics. :)